### DISCUSSION TITLE
Trouble installing Copilot in Visual Studio 2022

### DISCUSSION THREAD

**Discussion by dfeder06** [community/community#35726] [Copilot Conversations]
Hello.

I have been trying to install copilot in Visual Studio 2022 17.3.5.

I have installed the extension, but when I close and reopen the app, I never get the prompt to copy the code.

Does anyone know why this isn't working?

Thanks

**Comment by iamcjwilliams** [comment]
@antidextrous Can you provide more detail/ a screenshot of the error you're experiencing? Please try View -> Output; select GitHub Copilot from the pulldown

**Reply by dfeder06** [reply]
I figured it out on accident. Apparently, I needed to open up a new file for the popup to appear.

I already opened  up an old one before and it didn't work, but for some reason, when I made a new one, it worked.

Thanks
